src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.common;
+
+import java.util.*;
+import seedu.addressbook.common.Utils;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test; 
+
+public class UtilsTest {
+	
+	@Before public void setUp() {
```

Comment:
Good job on using `@Before`, but remember to start the method signature on a new line.